File: Loiane_Training_C#/Basico/D/Ex8.cs
using System;

class Ex8
{
    static void Main(string[] args)
    {
        int QUANTIDADE_DE_NUMEROS = 5;
        int sum = 0;
        for (int i = 0; i < QUANTIDADE_DE_NUMEROS; i++)
        {
            Console.Write("Digite um numero: ");
            int number = int.Parse(Console.ReadLine());

            sum += number;
        }

        Console.WriteLine("Soma: " + sum);
        Console.WriteLine("Media: " + (sum / QUANTIDADE_DE_NUMEROS));
    }
}

File: loiane-training-java/basico/Exer Aula 16_17/Ex8.java
import java.util.Scanner;

public class Ex8 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        double number = 0;
        double sum = 0;
        double average = 0;
        for (int i = 0; i < 5; i++) {
            System.out.println("Digite um numero");
            number = scan.nextInt();
            
            sum += number;
        }
        average = sum / 4;
        System.out.println("Soma:" + sum);
        System.out.println("Media:" + average);

        scan.close();
    }
}
